### DISCUSSION TITLE
Can't connect to any peer

### DISCUSSION THREAD

**Discussion by roxleopardo** [Chia-Network/chia-blockchain#5711] [Support]
I'm trying to use Chia 1.1.6 on my Mac (10.14.16) but it never found any peer. When I try to manually add any of the introducer's addresses my interface keeps loading for 30s and then nothing is added, so I saw this on my debug.log:

`2021-05-21T10:04:22.481 full_node chia.rpc.rpc_server     : WARNING  Error while handling message: Traceback (most recent call last):
  File "/Applications/chia-blockchain/chia/rpc/rpc_server.py", line 212, in safe_handle
    response = await self.ws_api(message)
  File "/Applications/chia-blockchain/chia/rpc/rpc_server.py", line 200, in ws_api
    return await f(data)
  File "/Applications/chia-blockchain/chia/rpc/rpc_server.py", line 161, in open_connection
    raise ValueError("Start client failed, or server is not set")
ValueError: Start client failed, or server is not set`

I have another Linux system on my network, I added his address and got my wallet synced thought him. But I want to sync with the official addresses too. On my network I use IPv4 and IPv6, so I forwarded port 8444 (IPv4) to my Linux system. And I forwarded port 8444 (IPv6) to all my IPv6 network block (I tested using ipv6scanner.com and its working). I tried to add IPv4 and IPv6 introducers IPs, but with no success. But I got sync using IPv4 and IPv6 address of my Linux system. Why I'm facing this problem with introducer's addresses?

OBS1: I'm having this problem with both versions DMG (Chia.app) and github (development).
OBS2: I already tried to re-install App and delete ˜/.chia folder (which includes deleting ˜/.chia/mainnet/db/peer_table_node.sqlite)

**Comment by BrandtH22** [comment]
@roxleopardo , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
